legend-engine-xt-relationalStore-pure/src/main/resources/core_relational/relational/sqlQueryToString/testSuite/selectSubClauses/windowColumn.pure

```diff
@@ -29,61 +29,68 @@ function <<paramTest.Test>> meta::relational::tests::dbSpecificTests::sqlQueryTe
 { 
   let func ={|meta::relational::tests::model::simple::Person.all()
     ->project([
-        col(p |$p.lastName,'lastName') 
+        col(p |$p.lastName,'lastName')
```

Comment:
I had to introduce ID in mapping to be able to sort expected output. Otherwise we could not guarantee integrity of tests when comparing arrays